client/html/layouts/basket/product/body-default.html

```diff
@@ -0,0 +1,114 @@
+<?php
```

Comment:
Right. So far it's all working again, except that I don't see anything. I also eddited the main-body-default.html, I'm not sure if this template is now being used or not? It's still located in layouts/basket/product/main-body-default.html